SU-HKKU/cardinal_service_profile
# Added padding around spotlight inline images

# READY FOR REVIEW

# Summary
- Added padding to images in the wysiwyg for spotlights.

# Need Review By (Date)
- soon

# Urgency
- medium

# Steps to Test
1. checkout this branch.
1. view the page `/stories/building-cultural-sensitivity-conservation` to see the inline image with a caption
1. validate there's spacing.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
